workers/unity/Assets/Generated/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Improbable.Gdk.Core.EditmodeTests")]
```

Comment:
Hmmm - we should rethink our test organisation if `Generated` needs to be visible to `Core.EditmodeTests`

Reply:
I think this should be resolved when/if MonoBehaviour support is out of core.

Other options:
- creating another test assembly for "things that depend on generated code"
- generating code into the test assembly? 🤔 - this sounds slightly worse

Reply:
Long term - creating a complete test project for codegen is the most attractive option for me. Once FPL lands this could tie in quite nicely

Reply:
It might make sense to have test components which have a good variety of features they use, then we could use those in tests instead of actual feature components.  It's also annoying atm that we don't even have any component in Core or the Playground with both events and commands on it atm, this could also help with that... (Though something like this might already exist in generated/gdk/tests in fact? I haven't looked much at those)